_benchmark/workloads/corpora.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: corpora
+parent: Workload reference
+nav_order: 70
+---
+
+The `corpora` element contains all the document corpora used by the workload. You can use document corpora across workloads by copying and pasting any corpora definitions. 
+
+## Example
+
+The following example defines a single document corpus with a count of `11658903` and an uncompressed size of `1544799789` bytes.
+
+```json
+  "corpora": [
+    {
+      "name": "movies",
+      "documents": [
+        {
+          "source-file": "movies-documents.json",
+          "document-count": 11658903, # Fetch document count from command line
+          "uncompressed-bytes": 1544799789 # Fetch uncompressed bytes from command line
+        }
+      ]
+    }
+  ]
+```
+
+## Configuration options
+
+Use the following options with `corpora`:
+
+Parameter | Required | Type | Description
+:--- | :--- | :--- | :---
+| `name` | Yes | String | The name the document corpus. Because OpenSearch Benchmark will use this name in its directories, use only lower case name without white spaces. |
+| `documents` | Yes | String | A list of document files. |
+| `meta` | No | String | A mapping of key-value pairs with additional metadata for a corpus. |
+
+
+The `documents` option consists of the following options:
+
+Parameter | Required | Type | Description
+:--- | :--- | :--- | :---
+| `source-file` | Yes | String | The file name to corresponding documents. When using Benchmark locally, documents are contained in a JSON file. When providing a `base_url`, use a compressed file format; `.zip`, `.bz2`, `.gz,` `.tar`, `.tar.gz`, `.tgz` or `.tar.bz2`. The compressed file must contain one JSON file with the name. |
+| `document-count` | Yes | Integer | The number of documents in the `source-file` that determines which client indexes correlate to which part of the document corpus (each N client get on N-th of the document corpus). When using a source that contains a document with a parent-child relationship, specify the number of parent documents. |
```

Comment:
Nit: "each N client get on N-th of the document corpus" -> "each N client gets an N-th of the document corpus"